src/styles/blocks/card-block.scss

```diff
@@ -0,0 +1,131 @@
+.container-sections {
+  display: grid;
+  gap: 48px 46px;
+  padding: 50px 40px;
+
+  grid-template-columns: repeat(1, 200px);
+  justify-content: center;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.container {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  // height: 408px;
+  padding: 32px 16px 16px;
+  font-family: $main-font-family;
+  font-style: $main-font-style;
+  color: $main-text-color;
+
+  &:hover {
+    transform: scale(120%);
```

Comment:
The transform property value 'scale(120%)' should be written as 'scale(1.2)' to adhere to the CSS specification.